pkg/client/application_categories_object_set.go

```diff
@@ -80,3 +80,5 @@ func buildApplicationCategoryObjectSet(response interface{}) []*nimbleos.Applica
 
 	return results
 }
+
+// List of supported actions on object sets
```

Comment:
Can we drop this unnecessary comment when there are no actions on the object set?